File: Python_Programming/numbers/check_greater.py
def ChkGreater(No1,No2):

    if(No1 > No2):
        print(No1)
    else:
        print(No2)

def main():

    iValue1 = 0
    iValue2 = 0

    print("Enter numbers : ")
    iValue1 = int(input())
    iValue2 = int(input())

    ChkGreater(iValue1,iValue2)

if __name__ == "__main__":
    main()

File: Cpp_programming/numbers/check_greater.cpp
// write a function which accepts two number and print the greater number 

#include<iostream>
using namespace std;

void ChkGreater(int iNo1, int iNo2)
{
    if(iNo1 > iNo2)
    {
        cout<<iNo1<<"\n";
    }
    else
    {
        cout<<iNo2<<"\n";
    }
}

int main()
{
    int iValue1 = 0, iValue2 = 0;

    cout<<"Enter number : \n";
    cin>>iValue1;
    cin>>iValue2;

    ChkGreater(iValue1,iValue2);

    return 0;
}